docs/en/developers/extensions/set-autoscale.md

```diff
@@ -3,6 +3,25 @@
 > Note: autoscale is one of the extension capabilities [installed from cap center](../cap-center.md),
 > please install it if you can't find it in `vela traits`.
 
+## Prerequisite
```

Comment:
We can auto-generate this section by `hack/references/generate.go`, like other sections of each `set-xxx.md` docs.

Reply:
Great idea, I think we can auto generate this section in the future : )

Reply:
yes, this is not high priority， we must finish it ASAP